WORKSPACE

```diff
@@ -39,7 +39,7 @@ bind(
 git_repository(
     name = "envoy",
     remote = "https://github.com/envoyproxy/envoy.git",
-    commit = "ec3446e9b3631568db6223e9eaf37fc6ce9f6cf2", # Sep 17, 2017 (added support for regex route matching)
+    commit = "d49bb3e967c16028520395efa56edf1af96f8a0e", # Sep 20, 2017 (bugfix for envoy crash w/ HTTP2 promise frames)
```

Comment:
this is a commit from envoy master 2h ago
it has
https://travis-ci.org/envoyproxy/envoy/builds/277913909?utm_source=github_status&utm_medium=notification
red builds; maybe we need to wait a bit to get it ?